docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+                                  'featureNodeHash': '59151c93',
+                                  'id': '2'},
+                                 {'attributes': [{'featureNodeHash': '2afbce16',
+                                                  'id': '3.1',
+                                                  'name': 'Text example',
+                                                  'required': False,
+                                                  'type': 'text'}],
+                                  'featureNodeHash': '8161c61a',
+                                  'id': '3'}],
+             'objects': [{'color': '#D33115',
+                          'featureNodeHash': '41d0a534',
+                          'id': '1',
+                          'name': 'cat',
+                          'shape': 'polygon'},
+                         {'color': '#FCDC00',
+                          'featureNodeHash': 'edf21536',
+                          'id': '2',
+                          'name': 'bus',
+                          'shape': 'polygon'},
+                         {'color': '#DBDF00',
+                          'featureNodeHash': '2a9ab898',
+                          'id': '3',
+                          'name': 'airplane',
+                          'shape': 'bounding_box'},
+                         {'color': '#A4DD00',
+                          'featureNodeHash': '56996de1',
+                          'id': '4',
+                          'name': 'bird',
+                          'shape': 'point'}]}
+
+
+Adding components to a project ontology
+---------------------------------------
+
+The ontology has two overall types of components, classifications and objects.
+Classifications relate to whole frames.
+For example, if want to be able to annotate if an image if a of a cat or a dog as a whole, you will add an animal classification with options "Cat" and "Dog."
+Objects, on the other hand, are specific to locations in a frame.
+For example, if you want to annotate where is the dog in the image, you would first add an object to the ontology.
+Objects could for example be a bounding box or a polygon indicating where the dog is.
+In the following subsections, we will show you how both scenarios are done.
+
+
+Adding a classification to a project ontology
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Here, we are in the "cat vs. dog" scenario from above.
+Adding classifications for cats and dogs to an ontology via the |sdk| is done as follows.
+
+The :meth:`add_classification() <.EncordClientProject.add_classification>` method takes the following parameters:
+
+* ``name`` the description of the classification
+* ``classification_type`` a value from the :class:`.ClassificationType` enum.
+* ``required`` - a boolean value specifying whether the classification is mandatory
+* ``options`` - a list of options that the annotator can choose from in case of :class:`CHECKLIST <.ClassificationType>`/:class:`RADIO <.ClassificationType>`.
```

Comment:
```suggestion
* ``options``: a list of options that the annotator can choose from in case of :class:`CHECKLIST <.ClassificationType>`/:class:`RADIO <.ClassificationType>`.
```